### ISSUE TITLE
Error 5 Compatiblity issue.

### ISSUE THREAD

**Issue by FightingBlaze** [Odie/skyui-vr#51]
There is a compatibility issue with these two files: 

quest_journal.swf
skyui/confipanel.swf

I don't know why these in particular. I am using the most latest file for both the vr version of skyrim script extender as well as skyui.

I tried removing the files manually, and the error message still showed up.

**Comment by Odie** [comment]
Howdy,

As far as I know, there hasn’t been any updates that would cause version warnings to pop up. You probably want to try and make sure:

- you’re actually launching the game via skse

- you have no other mods that are overriding skyui-vr

**Comment by FightingBlaze** [comment]
I found a solution. Turns out it was a swf file on the native skyrim vr data folder causing the issue. Once I deleted it, the ui worked perfectly. But thanks for getting back though.

Sent from Yahoo Mail on Android 
 
  On Sun, Mar 21, 2021 at 11:03 PM, Jonathan ***@***.***> wrote:   


Howdy,

As far as I know, there hasn’t been any updates that would cause version warnings to pop up. You probably want to try and make sure:
   
   -    
you’re actually launching the game via skse

   -    
you have no other mods that are overriding skyui-vr
